AmpegManualTestProject.txt

```diff
@@ -0,0 +1,172 @@
+
+
+Test Case 1
+
+Title 
+Creating an Account 
+
+Summary
+The user will be able to create an account and login to the website.
+
+Test objective
+To check whether A first time user of the website will be able to create an account or if a user that wants to create multiple accounts will be able to. 
+
+Steps Taken 
+1. Have the ampeg website open
+2. the header will consist of multiple words one of which is "shop", next to this will be a "person icon", click on the icon, it will open a pop up box 
+3. on the pop up box, click on "Create account" 
+4. the user will be redirected to a new page where they will be able to put their information in the feilds and then create their account by clicking "create your account". 
+5. the user will then be redirected to "account home" page. which is their profile page
+
+Expected result
+the user will be able to create an account
+
+Actual result
+the user was able to create an account and then automatically logged in the site and redirected to their profile page. 
+
+
+
+Test Case 2
+
+Title
+Contacting the Company
+
+Summary
+the user will be able to contact the company. 
+
+Test Objective
+To check whether a user who has a question about the website will be able to contact the company without any problems. 
+
+Steps taken
+1. be on the home page of the ampeg website 
+2. navigate to the bottom of the website using the scroll down slider on the right side of the screen. 
+3. on the bottom of the page there will be the term "company" in bold letter
+4. under which there will be "contact Us", click on this.
+5. the user will be redirected to another page, this page will have information about how they can contact the company. 
+
+Expected result
+the user will be able to contact the company by using the contact us page.
+
+Actual result
+the user will be able to contact the company by choosing whichever is best for them and also be able to give feedback through the contact us page.
+
+
+
+Test Case 3
+
+Title
+Ampeg History 
+
+Summary
+The user will be able to learn about ampeg's history
+
+Test Objective
+To test whether a user will be able to gather information about the history of ampeg 
+
+Steps taken
+1. be on ampeg homepage 
+2. at the very top of the website there will be multiple words written, one of which is "history"
+3. click on "history" 
+4. the user will be directed to another page where they will be able to see ampeg's history information. 
+5. at the bottom of the page, the user is also able to buy a book about ampeg which will have detailed information.
+6. at the bottom of the page click on amazon.com, it will be bold and underlined.
+7. user will be redirected to amazon where they will be able to buy a book. 
+
+Expected result
+the user is able to learn about the history 
+
+Actual result
+the user is able to learn about ampeg and if they want additional information they will be able to go to amazon and buy a book about ampeg.

```

Comment:
Actual Results should not be written in manual test cases. We'll cover execution later.